docs/Upgrade.md

```diff
@@ -1206,8 +1206,9 @@ The deprecated `<ThemeProvider theme>` prop was removed. Use the `ThemesContext.
 
 The `data-generator-retail` package has been updated to provide types for all its records. In the process, we renamed the `commands` resource to `orders`. Accordingly, the `nb_commands` property of the `customers` resource has been renamed to `nb_orders` and the `command_id` property of the `invoices` and `reviews` resources has been renamed to `order_id`.
 
+## `<DatagridBody>` No Longer Provides `record` Prop To `<DatagridRow>`
 
-
+The `<DatagridBody>` component no longer provides a `record` prop to its `<DatagridRow>` children. Instead, it provides a `recordContext` for each row. See the [`<Datagrid body/>`](./Datagrid.md#body) documentation to learn how to create your own row component.
```

Comment:
Pleae add a diff to show how the code must be changed (before: record injection, after: useRecordContext)